IdentityCore/src/broker_operation/request/account_request/MSIDBrokerOperationGetAccountsRequest.m

```diff
@@ -59,6 +59,12 @@ - (instancetype)initWithJSONDictionary:(NSDictionary *)json error:(NSError **)er
             }
             return nil;
         }
+        
+        _returnOnlySignedInAccounts = YES;
+        if(json[MSID_BROKER_SIGNED_IN_ACCOUNTS_ONLY_KEY])
+        {
+            _returnOnlySignedInAccounts = [json msidBoolObjectForKey:MSID_BROKER_SIGNED_IN_ACCOUNTS_ONLY_KEY];
```

Comment:
Should we fail if this key is not bool? Like we do for "succes" key:

```
 if (![json msidAssertTypeIsOneOf:@[NSString.class, NSNumber.class] ofKey:MSID_BROKER_OPERATION_RESULT_JSON_KEY required:YES error:error]) return nil;
_success = [json[MSID_BROKER_OPERATION_RESULT_JSON_KEY] boolValue];
```

Reply:
Since this parameter is optional, I think we should still fallback to default value and continue with the flow if the value is invalid. I have made some changes to the code to make sure the default value is YES for this parameter :)